index.js

```diff
@@ -2277,6 +2278,35 @@
       [$.FiniteNumber, $.FiniteNumber],
       function(a) { return a - 1; });
 
+  //# mult :: FiniteNumber -> FiniteNumber -> FiniteNumber
+  //.
+  //. Returns the product of two (finite) numbers.
+  //.
+  //. ```javascript
+  //. > S.mult(4, 2)
+  //. 8
+  //. ```
+  S.mult =
+  def('mult',
+      {},
+      [$.FiniteNumber, $.FiniteNumber, $.FiniteNumber],
+      function(a, b) { return a * b; });
+
+  //# div :: FiniteNumber -> NonZeroFiniteNumber -> FiniteNumber
+  //.
+  //. Returns the result of dividing its first argument (a finite number) by
+  //. its second argument (a non-zero finite number).
+  //.
+  //. ```javascript
+  //. > S.div(4, 2)
```

Comment:
The same logic applies here, actually. Let's use arguments which result in different values for `R.div` and `R.sub`. Also, let's make it clear that this is not integer division.